pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[tool.pyright]
+include = ["clients/python"]
+typeCheckingMode = "strict"
+venvPath = "."
+venv = ".venv"
```

Comment:
Use a uv-based venv in https://github.com/tensorzero/tensorzero/tree/main/clients/python.